openedx/features/edly/utils.py

```diff
@@ -233,3 +233,17 @@ def user_belongs_to_edly_organization(request, user):
         return True
 
     return False
+
+
+def edly_panel_user_has_edly_org_access(request):
+    """
+    Check if requesting user is an Edly panel user.
+    """
+    return EdlyUserProfile.objects.filter(
+        edly_sub_organizations__lms_site=request.site,
```

Comment:
This is bit suspecious `edly_sub_organizations__lms_site=request.site,`. `request.site` contains lms url on panel?